src/main/java/gtPlusPlus/core/recipe/RecipesMachines.java

```diff
@@ -956,12 +958,12 @@ private static void runModRecipes() {
         // Circuit programmer
         GTValues.RA.stdBuilder()
             .itemInputs(
-                ItemUtils.getSimpleStack(CI.robotArm_LV, 4 * (1)),
-                ItemList.Cover_Controller.get(1, CI.electricMotor_MV),
-                CI.machineHull_MV,
+                ItemList.Robot_Arm_LV.get(4),
+                ItemList.Cover_Controller.get(1, ItemList.Electric_Motor_MV.get(1)),
```

Comment:
This is for fallback behavior, but the controller cover should always be present so its not necessary
```suggestion
                ItemList.Cover_Controller.get(1),
```